examples/scenarioLambertSolver.py

```diff
@@ -0,0 +1,560 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2021, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
:small_red_triangle_down: Need to change the year to 2023, which is the year this file was created.